src/main/java/com/eerussianguy/blazemap/feature/maps/AtlasExporter.java

```diff
@@ -65,22 +74,38 @@ private static void export() {
                         }
```

Comment:
Given the issues Air is experiencing, I wonder if there should be extra per-tile error handling, so if the current region is borked, it just skips it and moves onto the next rather than erroring out the entire export?

Reply:
Actually, I think you said you have a fix for this in your working branch. Will leave it for when you push 😊

Reply:
No, this is everything from the working branch

Reply:
Those were done in the backport, since the issues have been happening in 1.18.2
Can you go ahead and take a look at that one?